webclient/src/md/api.md

```diff
@@ -1,24 +1,5 @@
 # API
 🛈 *Diese Seite ist aktuell nur auf Englisch verfügbar.*
 
-*The current API is under development and not yet stable.*
-
-The data of NavigaTUM can be accessed using an API, that is also used by NavigaTUM itself. This page provides a quick reference of the provided functions.
-
-## General
-The default endpoint is `https://api.to-be-determined.de`.
-To set the preferred language for the respone, set the `Accept-Language` to
-`de` or `en` accordingly.
-
-## Functions
-### get
-
-```
-GET <endpoint>/get/:id
-```
-
-### search
-
-```
-GET <endpoint>/search/:query
-```
+The data of NavigaTUM can be accessed using an API, that is also used by NavigaTUM itself.
```

Comment:
By removing this we have lost the hint, that our API is `under development and not yet stable`. Is this intentional?

Reply:
Added a note to this again. I just removed it there because I wanted to avoid having duplicate documentation.